pkgs/by-name/pe/perfect_dark/package.nix

```
@@ -0,0 +1,120 @@
+{
+  romID ? "ntsc-final",
+
+  lib,
+
+  fetchFromGitHub,
+  stdenv_32bit,
+
+  pkgsi686Linux, # SDL2 and zlib
+
+  pkg-config,
+  python3,
+}:
+let
+  roms = {
+    "ntsc-final" = {
+      binName = "pd";
+    };
+    "pal-final" = {
+      binName = "pd.pal";
+    };
+    "jpn-final" = {
+      binName = "pd.jpn";
+    };
```

```suggestion
    ntsc-final.binName = "pd";
    pal-final.binName = "pd.pal";
    jpn-final.binName = "pd.jpn";
```
Nit: Sets don't need to take this much space.